Sources/Private/Experimental/ExperimentalAnimationLayer.swift

```diff
@@ -138,7 +133,50 @@ final class ExperimentalAnimationLayer: CALayer {
 
 }
 
-// MARK: RootAnimationLayer
+// MARK: - CALayer + setupLayerHierarchy
+
+extension CALayer {
+  /// Sets up an `AnimationLayer` / `CALayer` hierarchy in this layer,
+  /// using the given list of layers.
+  func setupLayerHierarchy(
+    for layers: [LayerModel],
+    context: LayerContext)
+  {
+    var layersByIndex = [Int: CALayer]()
+
+    // First, we build the `AnimationLayer` / `CALayer` for each `LayerModel`
+    for layerModel in layers.reversed() {
```

Comment:
I think this is because the layer order in the json file is the reverse of the layer order expected by Core Animation (to get the expected Z ordering). Will double check and add a comment.